hal/armv7m/imxrt/117x/_init.S

```diff
@@ -181,34 +181,38 @@ _init_vectors:
 .type _start, %function
 _start:
 	cpsid if
-
+#if 0
```

Comment:
Why have you switched off `FLEX RAM` configuration? Comment will be helpful :)

Reply:
Done